README.md

```diff
@@ -1,13 +1,11 @@
 # Project Express API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I wish I had worked more on the styling, the looks are still very much WIP. I also had to remove the random book-function, I wish I could get that to work...
```

Comment:
What is WIP? I know the expression MVP but I've never heard of WIP :) 
Please share so I learn more! :)